scripts/build/package-builder.js

```diff
@@ -62,9 +62,10 @@ module.exports = {
  * This path should either be absolute or relative to the project root.
  * @param {boolean} enableIvy True, if Ivy should be used.
  * @param {string} description Human-readable description of the build.
+ * @param {boolean?} isRelease True, if the build should be stamped for a release.
  * @returns {Array<{name: string, outputPath: string}} A list of packages built.
  */
-function buildTargetPackages(destPath, enableIvy, description) {
+function buildTargetPackages(destPath, enableIvy, description, isRelease) {
```

Comment:
Nit: I generally prefer making it more explicit that an argument is optional (and also what the default value is) by providing the default value:

```suggestion
function buildTargetPackages(destPath, enableIvy, description, isRelease = false) {
```